app/helpers/spree/products_helper_decorator.rb

```diff
@@ -0,0 +1,10 @@
+Spree::ProductsHelper.module_eval do
+
+  def variant_full_price(variant)
+    product = variant.product
+    unless product.variants.active(current_currency).all? { |v| v.price == product.price && v.sale_price == product.price }
```

Comment:
@rishabhguptarishi how can **v.price == product.price && v.sale_price == product.price** be true? variant price and variant sale price will never be the same.